app.ts

```diff
@@ -0,0 +1,171 @@
+//#region AppLogic
+//#region API methods
+function getRecordCount() {
+    return fetch('http://localhost:2050/recordCount')
+        .then((response) => {
+            return response.json();
+        });
+}
+function getColumnNames() {
+    return fetch('http://localhost:2050/columns')
+        .then((response) => {
+            return response.json();
+        });
+}
+async function getRecords(fromID: number, toID: number) {
+    const request = 'http://localhost:2050/records?from=' + (fromID).toString() + '&to=' + (toID).toString();
+    return await fetch(request)
+        .then((response) => {
+            return response.json();
+        });
+}
+//#endregion
+
+//#region Data Loading methods
+
+function placeRecords(fromID: number, toID: number) {
+    getRecords(fromID, toID)
+        .then((records: Array<Array<string>>) => {
+            let appendable = '';
+            for (const record of records) {
+                 appendable += '<tr id="' + record[0] + '">';
+                for (const column of record) {
+                    appendable += '<td align="center">' + column.toString() + '</td>';     
+                }
+                appendable += '</tr>';
+                
+            }
+            $("#ContentBody").empty();
+            $("#ContentBody").append(appendable);
+        });
+    return [fromID as number, toID as number];
+}
+
+// Credit: https://gist.github.com/ca0v/73a31f57b397606c9813472f7493a940
+
+function placeRecordsFromCursor(cursor: Array<number>) {
+    cursor = cursor.sort((a,b) => {return a-b});
+    return placeRecords(cursor[0], cursor[1])
+}
+//#endregion
+
+//#region Handlers
+function getPageContent(fromID: number, toID: number) {
+    $("#HeaderRow").empty();
+    getColumnNames()
+        .then((columns: Array<string>) => {
+            for (const column of columns) {
+                let writable: string = '<th align="center">' + column + '</th>';
+                $("#HeaderRow").append(writable);
+            }
+        });
+    
+    return placeRecords(fromID, toID);
+}
+
+function toNumber(input: string | number, output: any = 1) {
+    if (typeof input === "string") {
+        if (output == 1) {
+            return parseInt(input);
+        } else {
+            return parseFloat(input);
+        }
+    }
+    else if (typeof input === "number") {
+        return input;
+    }
+    else {
+        return 0;
+    }
+}
+function nextPageResize(previousCursor: Array<number>): number {
+        let fromID = toNumber(previousCursor.sort((a, b) => {return a - b})[0]);
```

Comment:
Most of your `let` variables should be `const`. `let` signifies that you are making a variable that could be adjusted later down the line. In the case of an array or an object a `const` signifies that the variable is not going to be reassigned to a completely new object or array.